config/routes.rb

```diff
@@ -3,13 +3,14 @@
   devise_for :users
 
   resources :parks, only: [:index, :show, :new, :create, :edit, :update, :destroy] do
-    resources :reviews, only: [:index, :new, :create, :edit]
+    resources :reviews, only: [:index, :new, :show, :create, :destroy]
```

Comment:
While you don't NEED to update it, at this point you might want to change this to use the shorter except instead. Also, I don't think `destroy` is a part of this PR, so it shouldn't yet be added here